rootfs/etc/cont-init.d/08-svc-snmptrapd.sh

```diff
@@ -2,6 +2,15 @@
 
 SIDECAR_SNNMPTRAPD=${SIDECAR_SNNMPTRAPD:-0}
 LIBRENMS_SNMP_COMMUNITY=${LIBRENMS_SNMP_COMMUNITY:-librenmsdocker}
+SNMP_ACTION=${SNMP_ACTION:-log,execute,net}
```

Comment:
To align with net-SNMP, `ACTION` should be renamed `PROCESSING_TYPE`

```suggestion
SNMP_PROCESSING_TYPE=${SNMP_PROCESSING_TYPE:-log,execute,net}
```